src/HttpServerFactory.php

```diff
@@ -11,7 +11,7 @@
 
 use Psr\Container\ContainerInterface;
 use Swoole\Http\Server as SwooleHttpServer;
-use SwooleRuntime;
+use Swoole\Runtime as SwooleRuntime;
```

Comment:
Is this true across all versions of Swoole?

Reply:
Yes, I saw all the changes in the [`swoole_runtime.cc`](https://github.com/swoole/swoole-src/commits/master/swoole_runtime.cc) file.